.github/workflows/container-image.yml

```diff
@@ -1,5 +1,7 @@
-name: build container image
+name: Build container image
 on:
+  # User can trigger this workflow in the Github UI
+  workflow_dispatch:
```

Comment:
I would drop this line, for other github workflows is ok, but for releases we shouldn't overwrite and re-release them.